spec/controllers/games_controller_spec.rb

```diff
@@ -58,5 +58,16 @@
     end  
   end
 
+  describe "games#update action"  do
+    it "should allow users to join a game as player 2" do
+      user = FactoryGirl.create(:user)
+      sign_in user
+      g = FactoryGirl.create(:game, sith_user_id: user.id) 
+
+      patch :update, id: g.id, jedi_user_id: user.id
```

Comment:
We should also have a test for the `sith_user_id`, because we have an `if` statement and you always want to test both halfs of the statement.